Makefile

```diff
@@ -12,7 +12,7 @@ depend_log    = $(build_dir)/.depend
 build_log     = $(build_dir)/.build
 merge_log     = $(build_dir)/.merge
 
-cluster_avsc_link  = "https://raw.githubusercontent.com/snowplow/iglu-central/master/schemas/com.snowplowanalytics.dataflowrunner/ClusterConfig/avro/1-0-0"
+cluster_avsc_link  = "https://raw.githubusercontent.com/snowplow/iglu-central/master/schemas/com.snowplowanalytics.dataflowrunner/ClusterConfig/avro/1-1-0"
```

Comment:
This should be:

    http://iglucentral.com/schemas/com.snowplowanalytics.dataflowrunner/ClusterConfig/avro/1-1-0